model/load_balancer.rb

```diff
@@ -38,7 +38,7 @@ def certificate_bundle_path
   def add_vm(vm)
     DB.transaction do
       super
-      incr_update_load_balancer
+      Strand.create_with_id(schedule: Time.now, prog: "Vnet::LoadBalancerHealthProbes", label: "health_probe", stack: [{subject_id: id, vm_id: vm.id}])
```

Comment:
No need to give `schedule: Time.now` parameter. The default is already `Time.now` in DB